ci-operator/step-registry/kubevirt/install/kubevirt-install-commands.sh

```diff
@@ -29,7 +29,11 @@ elif [ -n "${CNV_PRERELEASE_CATALOG_IMAGE}" ]
   then
   CNV_RELEASE_CHANNEL=stable
 else
-  CNV_RELEASE_CHANNEL=nightly-$(ocp_version)
+  if [ "${CNV_PRERELEASE_LATEST_CHANNEL}" == "true" ]; then
```

Comment:
why not make this the actual channel name or version string so you can tweak it to other values
in the future if needed?

default would be ""

so, code would be something like:

if [ "${CNV_PRERELEASE_LATEST_CHANNEL_VERSION}" != "" ]; then
 CNV_RELEASE_CHANNEL=nightly-${CNV_PRERELEASE_LATEST_CHANNEL_VERSION}
  else
    CNV_RELEASE_CHANNEL=nightly-$(ocp_version)
  fi

Reply:
We don't want to test wrong channel versions at master branch, also this is a very temporal knob that will be removed after stuff settle up at kubevirt.

Reply:
ok, sounds like you think it's overkill then.

/lgtm